tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/MemberList.kt

```diff
@@ -0,0 +1,90 @@
+package net.corda.cli.plugins.network
+
+import net.corda.cli.plugins.common.RestClientUtils.createRestClient
+import net.corda.cli.plugins.common.RestCommand
+import net.corda.membership.rest.v1.MemberLookupRestResource
+import net.corda.membership.rest.v1.types.response.RestMemberInfo
+import picocli.CommandLine
+
+@CommandLine.Command(name = "members", description = ["List members"])
+class MemberList : RestCommand(), Runnable {
+    override fun run() {
+        // Call the `getMembersList` function with default or empty values for the options
+        getMembersList(
+            holdingIdentityShortHash = null,
+            commonName = null,
+            organizationUnit = null,
+            organization = null,
+            locality = null,
+            state = null,
+            country = null
+        )
+    }
+    @Suppress("LongParameterList")
+    @CommandLine.Command(
+        name = "members-list",
+        description = ["Shows the list of members on the network."]
+    )
+    fun getMembersList(
```

Comment:
This doesn't need `Command` any more, we can change it to -

```suggestion
    private fun getMembersList() {
```

Reply:
I have addressed the change in the latest commit